dspot-maven/pom.xml

```diff
@@ -48,10 +57,17 @@
 			<artifactId>dspot</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+
+		<dependency>
+			<groupId>eu.stamp-project</groupId>
+			<artifactId>dspot</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
Duplicated dependency to DSpot, right?

Reply:
No it's not a duplicated dependency. Dspot Maven plugin needs the DSpot dependency.

Reply:
I meant in the pom, there is exactly the same dependency just above, _i.e._ dependency to **DSpot**, isn't?